src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age
+    && (
+    <p className="Person__age">
+      {`I am ${person.age}`}
+    </p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+        : 'I am not married'}
```

Comment:
All right, but if you have complicated conditions then it is better to move it out of JSX, example: 

```
   const getMarriedStatus = () => {
     if (isMarried) {
       return `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
     }
     
     retrun 'I am not married';
   }
```
and then in your jsx:

```
<>
....
  {getMarriedStatus()}
....
</>
```